File: app/src/main/java/com/example/jy/jyweather/entity/WeatherBean.java
package com.example.jy.jyweather.entity;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * 和风天气实体类
 * <p>
 * Created by Yang on 2017/10/15.
 */
public class WeatherBean {

    private String status;

    private UpdateBean update;

    private BasicBean basic;

    private NowBean now;

    @SerializedName("daily_forecast")
    private List<DailyForecastBean> dailyForecasts;

    @SerializedName("hourly")
    private List<HourlyForecastBean> hourlyForecasts;

    @SerializedName("lifestyle")
    private List<LifestyleBean> lifestyles;

    public String getStatus() {
        return status;
    }

    public UpdateBean getUpdate() {
        return update;
    }

    public BasicBean getBasic() {
        return basic;
    }

    public NowBean getNow() {
        return now;
    }

    public List<DailyForecastBean> getDailyForecasts() {
        return dailyForecasts;
    }

    public List<HourlyForecastBean> getHourlyForecasts() {
        return hourlyForecasts;
    }

    public List<LifestyleBean> getLifestyles() {
        return lifestyles;
    }
}

File: app/src/main/java/com/jy/weather/entity/WeatherBean.kt
package com.jy.weather.entity

import com.google.gson.annotations.SerializedName

/**
 * 和风天气实体类
 *
 * Created by Yang on 2017/10/15.
 */
data class WeatherBean(
    val status: String,

    val update: UpdateBean,

    val basic: BasicBean,

    val now: NowBean,

    @SerializedName("daily_forecast")
    val dailyForecasts: List<DailyForecastBean>,

    @SerializedName("hourly")
    val hourlyForecasts: List<HourlyForecastBean>,

    @SerializedName("lifestyle")
    val lifestyles: List<LifestyleBean>
)